go.mod

```diff
@@ -7,10 +7,10 @@ require (
 	github.com/giantswarm/appcatalog v0.1.11
 	github.com/giantswarm/apprclient v0.2.0
 	github.com/giantswarm/backoff v0.2.0
-	github.com/giantswarm/e2esetup v0.2.0
+	github.com/giantswarm/e2esetup v0.2.1-0.20200401192455-acfdbff0dabe
 	github.com/giantswarm/errors v0.2.0
 	github.com/giantswarm/exporterkit v0.2.0
-	github.com/giantswarm/helmclient v0.2.0
+	github.com/giantswarm/helmclient v0.0.0-20200331110847-d697c5c4a0fb
```

Comment:
Can you try updating the deps again with our tags? The time/hash prefix should not be necessary with out libraries.

Reply:
This is because it's using the helm3 branch of helmclient. Before this gets merged to master we'll merge helmclient into master and create a tag.

Reply:
This is because we are using `helm3` branch in helmclient for this branch.